sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+from xml.etree.ElementTree import ElementTree
+
+from azure.core.async_paging import AsyncItemPaged
+from azure.servicebus.aio.management._utils import extract_data_template, get_next_template
+from azure.core.exceptions import ResourceNotFoundError
+from azure.core.pipeline import AsyncPipeline
+from azure.core.pipeline.policies import HttpLoggingPolicy, DistributedTracingPolicy, ContentDecodePolicy, \
+    RequestIdPolicy, AsyncBearerTokenCredentialPolicy
+from azure.core.pipeline.transport import AioHttpTransport
+
+from ...management._generated.models import QueueDescriptionFeed, TopicDescriptionEntry, \
+    QueueDescriptionEntry, SubscriptionDescriptionFeed, SubscriptionDescriptionEntry, RuleDescriptionEntry, \
+    RuleDescriptionFeed, NamespacePropertiesEntry, CreateTopicBody, CreateTopicBodyContent, \
+    TopicDescriptionFeed, CreateSubscriptionBody, CreateSubscriptionBodyContent, CreateRuleBody, \
+    CreateRuleBodyContent, CreateQueueBody, CreateQueueBodyContent, \
+    QueueDescription as InternalQueueDescription, TopicDescription as InternalTopicDescription, \
+    SubscriptionDescription as InternalSubscriptionDescription, RuleDescription as InternalRuleDescription, \
+    NamespaceProperties
+
+from ..._common.utils import parse_conn_str
+from ..._common.constants import JWT_TOKEN_SCOPE
+from ...aio._base_handler_async import ServiceBusSharedKeyCredential
+from ...management._generated.aio._configuration_async import ServiceBusManagementClientConfiguration
+from ...management._generated.aio._service_bus_management_client_async import ServiceBusManagementClient \
+    as ServiceBusManagementClientImpl
+from ...management import _constants as constants
+from ._shared_key_policy_async import AsyncServiceBusSharedKeyCredentialPolicy
+from ...management._models import QueueRuntimeInfo, QueueDescription, TopicDescription, TopicRuntimeInfo, \
+    SubscriptionDescription, SubscriptionRuntimeInfo, RuleDescription
+from ...management._xml_workaround_policy import ServiceBusXMLWorkaroundPolicy
+from ...management._handle_response_error import _handle_response_error
+from ...management._model_workaround import avoid_timedelta_overflow
+
+
+if TYPE_CHECKING:
+    from azure.core.credentials_async import AsyncTokenCredential  # pylint:disable=ungrouped-imports
+
+
+class ServiceBusManagementClient:  #pylint:disable=too-many-public-methods
+    """Use this client to create, update, list, and delete resources of a ServiceBus namespace.
+
+    :param str fully_qualified_namespace: The fully qualified host name for the Service Bus namespace.
+    :param credential: To authenticate to manage the entities of the ServiceBus namespace.
+    :type credential: Union[TokenCredential, ServiceBusSharedKeyCredential]
+    """
+
+    def __init__(self, fully_qualified_namespace, credential, **kwargs):
+        # type: (str, Union[AsyncTokenCredential, ServiceBusSharedKeyCredential], Dict[str, Any]) -> None
```

Comment:
Very naive question: I thought we could use inline typing for AIO?

Reply:
Yeah - we should use Python3 syntax where we can

Reply:
You're right. I omitted this one. Good catch.

Reply:
Updated